modules/nf-core/limma/differential/tests/main.nf.test

```diff
@@ -72,6 +72,130 @@ nextflow_process {
             )
         }
 
+    }
+
+        test("test_limma_differential - null formula and null complex contrasts") {
```

Comment:
Needs an indent fix. Sorry if I'm not seeing it, but isn't this the same as one of the tests above?